proxy/connection.go

```diff
@@ -59,16 +58,11 @@ func (c *connection) processMessages() {
 }
 
 func (c *connection) handleMessage() (err error) {
-	isMaster := false
-	var reqOpCode, resOpCode wiremessage.OpCode
+	var tags []string
 
 	defer func(start time.Time) {
-		_ = c.statsd.Timing("handle_message", time.Since(start), []string{
-			fmt.Sprintf("success:%v", err == nil),
-			fmt.Sprintf("is_master:%v", isMaster),
-			fmt.Sprintf("request_op_code:%v", reqOpCode),
-			fmt.Sprintf("response_op_code:%v", resOpCode),
```

Comment:
Why do we not want to store the response op code in our tags anymore?

Reply:
thanks, fixed